android/src/main/java/com/getcapacitor/community/intercom/intercom/MessagingService.java

```diff
@@ -0,0 +1,29 @@
+package com.getcapacitor.community.intercom.intercom;
```

Comment:
this MessagingService could be implemented in the MainActivity if using the app's AndroidManifest